Log historical view - new parameters (#7160)
DataDog/documentation: content/en/logs/archives/rehydrating.md

5. (Optional) **Add a description** to give your team context about the purpose of the historical view.

6. Define the maximum number of logs that should be rehydrated in this historical view, **from 1 million to 1 billion**

> 7. (Optional) Define the retention of those rehydrated logs. Choose the `Decide Later` option if you don't know yet how long you are going to need them.

Reviewer: ```suggestion
7. (Optional) Define the retention period of those rehydrated logs. Choose the `Decide Later` option if you don't know yet how long you are going to need them.
```

> 7. (Optional) Define the retention period of those rehydrated logs. Choose the `Decide Later` option if you don't know yet how long you are going to need them.

Reviewer 2: ```suggestion
7. (Optional) Define the retention period of the rehydrated logs. Choose `Decide Later` if you are unsure how long you will need the logs.
```
